### PR TITLE
changed the playtime math to fix 1 weeks, etc.

### PR DESCRIPTION
**Pull request by DylanBeMe**
changed the basic playtime math so it doesn't show 1 minutes, 1 weeks and 1 months.

### CODE REVIEW

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5468353&bid=17922255)

File: src/app/desktop/components/Instances/Instance.js [hunk 1]
```diff
@@ -177,12 +177,32 @@
...
+        return `${min} minutes`;
+      }
+      if (min < 2) {
+        return `${min} minute`;
+      }
+      else {
+        return `${min} minutes`;
+      }
+    }
+     if (weeks < 4) {
```
Comment (killpowa) [inline src/app/desktop/components/Instances/Instance.js:190]: Codacy Issue found: [Delete `·`](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5468353)

File: src/app/desktop/components/Instances/Instance.js [hunk 2]
```diff
@@ -177,12 +177,32 @@
...
+      if (weeks < 2) {
+        return `${weeks} week`;
+      }
+    }
+    if (weeks > 1) {
+      return `${weeks} weeks`;
     }
-    if (months > 0) {
-      return `${months} months`;
+    else {
```
Comment (killpowa) [inline src/app/desktop/components/Instances/Instance.js:198]: Codacy Issue found: [Unnecessary 'else' after 'return'.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5468353)

File: src/app/desktop/components/Instances/Instance.js [hunk 3]
```diff
@@ -164,6 +164,44 @@ const Instance = ({ instanceName }) => {
 
   const isPlaying = startedInstances[instanceName];
 
+  const calcTime = minutes => {
```
Comment (killpowa) [inline src/app/desktop/components/Instances/Instance.js:167]: Codacy Issue found: ['calcTime' is assigned a value but never used.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5468353)

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- src/app/desktop/components/Instances/Instance.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5468353&bid=17922255)

File: src/app/desktop/components/Instances/Instance.js (hunk 3 above)
Comment (DylanHameleers) [reply src/app/desktop/components/Instances/Instance.js:167]: How tho? I didn't change that..

**Pull request closed without merging**